package.json

```diff
@@ -14,19 +16,22 @@
   },
   "license": "MIT",
   "scripts": {
+    "build": "rollup -c rollup.config.js",
+    "prepare": "npm run build",
     "test": "mocha",
     "test:test262": "node run_test262.js",
     "lint": "eslint -c .eslintrc.json ."
   },
   "peerDependencies": {
     "acorn": "^6 || ^7"
```

Comment:
If we remove `require` altogether, `acorn` is probably no longer a peer dependency. But I still don't know if we want to do that or leave conditional `require` for CJS.

Reply:
done